docs/source/extensions/logging.rst

```diff
@@ -14,56 +14,85 @@
 Logging
 #######
 
-Lightning supports the most popular logging frameworks (TensorBoard, Comet, etc...).
+Supported Loggers
+=================
+
+The following are loggers we support:
+
+.. currentmodule:: pytorch_lightning.loggers
+
+.. autosummary::
+    :toctree: generated
+    :nosignatures:
+    :template: classtemplate.rst
+
+    CometLogger
+    CSVLogger
+    MLFlowLogger
+    NeptuneLogger
+    TensorBoardLogger
+    TestTubeLogger
+    WandbLogger
 
-By default, Lightning uses `PyTorch TensorBoard <https://pytorch.org/docs/stable/tensorboard.html>`__ logging  under the hood, and stores the logs to a directory (by default in ``lightning_logs/``).
+
+The above loggers will normally plot an additional chart (**global_step VS epoch**). Depending on the loggers you use, there might be some additional charts too.
+
+By default, Lightning uses ``TensorBoard`` logger under the hood, and stores the logs to a directory (by default in ``lightning_logs/``).
 
 .. testcode::
 
     from pytorch_lightning import Trainer
 
-    # Automatically logs to a directory
-    # (by default ``lightning_logs/``)
+    # Automatically logs to a directory (by default lightning_logs/)
     trainer = Trainer()
 
 To see your logs:
 
 .. code-block:: bash
 
+    # Install tensorboard
+    pip install tensorboard
```

Comment:
Isn't it installed by default for Lightning? I don't think the installment command is necessary